src/backend/TrafficCourts/Citizen.Service/Services/Impl/CitizenDocumentService.cs

```diff
@@ -105,11 +105,13 @@ public async Task<List<FileMetadata>> FindFilesAsync(DocumentProperties properti
             FileMetadata fileMetadata = new()
             {
                 FileId = result.Id,
-                FileName = result.FileName,
-                DocumentType = properties.DocumentType,
-                NoticeOfDisputeGuid = properties.NoticeOfDisputeId?.ToString("d"),
-                VirusScanStatus = properties.VirusScanStatus,
-                DocumentStatus = properties.StaffReviewStatus,
+                //FileName = result.FileName, // this is always null;
+                FileName = properties.DocumentName,
+                DocumentType = properties.DocumentType,
+                DocumentSource = properties.DocumentSource,
+                NoticeOfDisputeGuid = properties.NoticeOfDisputeId?.ToString("d"),
+                VirusScanStatus = properties.VirusScanStatus,
+                DocumentStatus = properties.StaffReviewStatus,
             };
 
             fileData.Add(fileMetadata);

```

Comment:
I think it is better to filter the DocumentSource here, so frontend will never receive the staff documents. And probably need to block the access/edit/delete to those files in citizen api